app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,27 @@
+# Greetings Controller for API
+module Api
+  module V1
+    # Greetings Controller for API
+    class GreetingsController < ApplicationController
+      def index
+        # generate random number between 1 and 5
+        random_number = rand(1..5)
+
+        # return a random greeting
+        @greeting = Greeting.find(random_number)
+
+        # return json response
+        return json_response(@greeting) if @greeting
+
+        # return error if no greeting found
+        json_response({ message: 'No Greeting Found' }, :not_found)
+      end
+
+      private
+
+      def json_response(object, status = :ok)
+        render json: object, status:
+      end
+    end
+  end
```

Comment:
- Hey, On your frontend, the greeting doesn't seem to be displaying, please, kindly fix this to make your app run as expected.